go/vt/vtgate/engine/route.go

```diff
@@ -584,27 +587,26 @@ func (route *Route) sort(in *sqltypes.Result) (*sqltypes.Result, error) {
 		InsertID:     in.InsertID,
 	}
 
+	comparers := extractSlices(route.OrderBy)
```

Comment:
why are we creating this at run-time? couldn't this be done at plan time?

Reply:
We are doing this on run-time since these compares have to created for all engines again and cannot be shared by the plans